rest-parameters-es6/index.html

```diff
@@ -0,0 +1,83 @@
+---
+feature_name: Rest parameters (ES2015)
+chrome_version: 47
+feature_id: 5937087912083456
+---
+
+<h3>Background</h3>
+<p>
+Rest parameters allow your functions to have variable number of arguments without using the arguments object. The rest parameter is an instance of Array so all the array methods just work.
+</p>
```

Comment:
Can you mention in the Background section that there can only be a single rest parameter for a given function? It's obvious when you know about the feature, but I could imagine some folks newer folks wondering what happens if you try `function foo(...first, ...second)`